index.html

```diff
@@ -1528,6 +1532,26 @@ <h2>Relationship to Verifiable Credentials</h2>
 is the controller of the <a>verification method</a> used to verify
 the proof.
         </p>
+
+        <p>
+Document authors and implementers are advised to understand the difference
+between the validity period of a <a href="#proofs">proof</a>, which is expressed
+using the <a href="#dfn-created">`created`</a> and <a
+href="#dfn-expires">`expires`</a> properties, and the validity period of a
+<a>verifiable credential</a>, which is expressed using the
+<a data-cite="?VC-DATA-MODEL-2.0#defn-validFrom">`validFrom`</a> and
+<a data-cite="?VC-DATA-MODEL-2.0#defn-validUntil">`validUntil`</a> properties.
+While these properties might sometimes express the same validity periods, at
+other times they might not be aligned. When verifying a
+<a href="#proofs">proof</a>, it is important to ensure that the time is within
+the validity period for the proof. When
+<a data-cite="?VC-DATA-MODEL-2.0#validation">validating</a>
+a <a>verifiable credential</a>, it is important to ensure that the time is
```

Comment:
Trying to indicate that "the time" one is whatever is of interest, which may commonly be the current time, but does not have to be.

```suggestion
<a href="#proofs">proof</a>, it is important to ensure that the time of interest
(the current time or another time) is within the validity period for the proof.
When <a data-cite="?VC-DATA-MODEL-2.0#validation">validating</a> a
<a>verifiable credential</a>, it is important to ensure that the time of interest is
```